src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final String ROW_SUPPLY = "supply";
+    static final String ROW_BUY = "buy";
+    static final String ROW_RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
getStatistic method should have 3 lines - 3 private methods calls
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/153b2a58-0e2d-4f54-aed3-94e2d1f10f6b)

Reply:
I`m sorry was inattentive